src/pages/PoolsPage/v3/SupplyLiquidityV3/index.tsx

```diff
@@ -205,102 +197,52 @@ export function SupplyLiquidityV3() {
     [handleCurrencySelect],
   );
 
-  const handleCurrencySwap = useCallback(() => {
-    // history.push(`/add/${currencyIdB}/${currencyIdA}`);
-    resetState();
-  }, [handleCurrencySelect, currencyIdA, currencyIdB]);
-
-  const handlePopularPairSelection = useCallback((pair: [string, string]) => {
-    const WMATIC = '0x0d500b1d8e8ef31e21c99d1db9a6444d3adf1270';
-    // history.push(
-    //   `/add/${pair[0] === WMATIC ? 'MATIC' : pair[0]}/${
-    //     pair[1] === WMATIC ? 'MATIC' : pair[1]
-    //   }`,
-    // );
-    resetState();
-  }, []);
-
-  // const handleStepChange = useCallback(
-  //   (_step) => {
-  //     history.push(`/add/${currencyIdA}/${currencyIdB}/${_step}`);
-  //   },
-  //   [currencyIdA, currencyIdB, history],
-  // );
+  // maybe needed when we need function to swap tokens when supplying liquidity
```

Comment:
Why was this commented out

Reply:
This is for swap tokens when supplying new liquidity which is not used for now. But I think we can use it later.